### PR TITLE
ccheck

### PR DESCRIPTION
**Pull request by Sonal2808**
Please check now. I added About in footer.js and it was not working like that so added in App.js too.

### CODE REVIEW

File: roomranger/frontend/src/App.css [hunk 1]
```diff
@@ -11,27 +11,74 @@
...
+ right:0;
  width:100%;
  height: 50px;
- /* background-color: rgb(236, 229, 229); */
+ /*background-color: rgb(236, 229, 229);*/
  text-align: center;
  color: white;
 }
 
 .nbar{
```
Comment (LunaLiu829) [inline roomranger/frontend/src/App.css]: i think you should command this line too.

File: roomranger/frontend/src/App.js [hunk 2]
```diff
@@ -1,81 +1,95 @@
-import './App.css';
-import "../node_modules/bootstrap/dist/css/bootstrap.min.css"
+import "./App.css";
```
Comment (LunaLiu829) [inline roomranger/frontend/src/App.js]: To maintain consistency, let's all use single quotation marks.

File: roomranger/frontend/src/App.js [hunk 3]
```diff
@@ -1,81 +1,95 @@
...
+                      <Route path='/' element={<Home />} />
+                      <Route path='login' element={<LoginPage />} />
+                      <Route path='register' element={<Register />} />
+                   </Route> */}
+                  {/* </Route> */}
+                  {/* <Route path='/' element={<RouteHome />}> */}
+                    <Route path ='/'element={<Home />} />
+                    <Route path='Register' element={<Registration />} />
+                    <Route path='login' element={<LoginPage />} />
+                    <Route path='about' element={<About />} />
```
Comment (LunaLiu829) [inline roomranger/frontend/src/App.js:58]: How is About show now? Please send us a screenshot! Will it show in everypage?

**Review by LunaLiu829 — APPROVED** [review]
Good job.

File: roomranger/frontend/src/App.css (hunk 1 above)
Comment (LunaLiu829) [reply roomranger/frontend/src/App.css]: line 24

Comment (Sonal2808) [reply roomranger/frontend/src/App.css]: Done

File: roomranger/frontend/src/App.js (hunk 2 above)
Comment (Sonal2808) [reply roomranger/frontend/src/App.js]: Done

File: roomranger/frontend/src/App.js (hunk 3 above)
Comment (Sonal2808) [reply roomranger/frontend/src/App.js:58]: sure

File: roomranger/frontend/public/images/sonal.jpg
Comment (sammu1991) [inline roomranger/frontend/public/images/sonal.jpg:1]: Golden lady is still here :)

File: roomranger/frontend/src/index.css
Comment (sammu1991) [inline roomranger/frontend/src/index.css:1]: could you please try moving background url to light theme body and see if it works?

File: roomranger/src/main/java/org/launchcode/roomranger/models/User.java
Comment (sammu1991) [inline roomranger/src/main/java/org/launchcode/roomranger/models/User.java:1]: There is no need of separate confirm password column in user table..

**Review by sammu1991 — APPROVED** [review]

File: roomranger/frontend/src/index.css
Comment (Sonal2808) [reply roomranger/frontend/src/index.css:1]: Sure. I would

File: roomranger/src/main/java/org/launchcode/roomranger/models/User.java
Comment (Sonal2808) [reply roomranger/src/main/java/org/launchcode/roomranger/models/User.java:1]: removed

Comment (Sonal2808) [reply roomranger/src/main/java/org/launchcode/roomranger/models/User.java:1]: Done

**Pull request merged**